content/dtolnay trick.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Dtolnay trick"
++++
+
+See [**semver trick**](#semver-trick).
```

Comment:
¹ This link became broken. It might be a good idea to implement a `see_also` attribute for these entries, but for the time being, tweaking this to point to the right page or fragment would be enough.